app/main.py

```diff
@@ -1,25 +1,68 @@
+from __future__ import annotations
 from abc import ABC
+from typing import Union
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: IntegerRange, name: str) -> None:
+        self._protected_name = "_" + name
+
+    def __get__(self, instance: callable, owner: IntegerRange) -> None:
+        getattr(instance, self._protected_name)
+
+    def __set__(self, instance: callable, value: int) -> None:
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError
+        setattr(instance, self._protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self,
+                 name: str,
+                 age: int,
+                 weight: int,
+                 height: int
+                 ) -> None:
```

Comment:
you cat write like this
`    def __init__(self, name: str, age: int, weight: int, height: int) -> None:`
or 
```
    def __init__(
            self, 
            name: str, 
            age: int, 
            weight: int, 
            height: int
    ) -> None:
```